util/cloud/gcp/gcs.cc

```diff
@@ -35,6 +35,11 @@ auto Unexpected(std::errc code) {
 
 const char kInstanceTokenUrl[] = "/computeMetadata/v1/instance/service-accounts/default/token";
 
+const char gcloudCliDir[] = "~/.config/gcloud/";
```

Comment:
notation for constants is `kCloudCliDir`. here and below

Reply:
IMO `kGcloudCliDir` makes more sense (emphasis on `gcloud` since it's specifically google's CLI, not something generic)